shark-graph/src/main/java/shark/FieldValuesReader.kt

```diff
@@ -21,8 +19,11 @@ import shark.ValueHolder.IntHolder
 import shark.ValueHolder.LongHolder
 import shark.ValueHolder.ReferenceHolder
 import shark.ValueHolder.ShortHolder
+import shark.internal.readInt
+import shark.internal.readLong
+import shark.internal.readShort
 
-internal class FieldValuesReader(
+class FieldValuesReader(
```

Comment:
Why does this need to be public?

Reply:
I figured out why and not super happy with us exposing this which isn't really a great API.